app/views/hub/metrics/index.html.erb

```diff
@@ -21,79 +21,172 @@
             <td class="index-table__cell" style="padding-left: 60px; font-style: italic;"><%= vita_partner.name %></td>
             <td class="index-table__cell attention-needed-breach"  data-js-count=<%= @report.attention_needed_breaches[vita_partner.id] %>>
               <%= @report.attention_needed_breaches[vita_partner.id] %>
             </td>
+            <td class="index-table__cell attention-needed-breach-percentage">
+              <%= @report.active_sla_clients_count[vita_partner.id] %>
+
+            </td>
             <td class="index-table__cell communication-breach" data-js-count=<%= @report.communication_breaches[vita_partner.id] %>>
               <%= @report.communication_breaches[vita_partner.id]%>
             </td>
+            <td class="index-table__cell communication-breach-percentage">
+              <%= @report.active_sla_clients_count[vita_partner.id] %>
+
+            </td>
             <td class="index-table__cell interaction-breach" data-js-count=<%= @report.interaction_breaches[vita_partner.id] %>>
               <%= @report.interaction_breaches[vita_partner.id] %>
             </td>
+            <td class="index-table__cell interaction-breach-percentage">
+              <%= @report.active_sla_clients_count[vita_partner.id] %>
+
+            </td>
           </tr>
 
           <% vita_partner.child_sites.each do |site| %>
-            <tr class="index-table__row site">
+            <tr class="index-table__row site" data-js-count=<%= @report.active_sla_clients[site.id] %>>
               <td class="index-table__cell" style="padding-left: 60px"><%= site.name %> </td>
+
               <td class="index-table__cell attention-needed-breach" data-js-count=<%= @report.attention_needed_breaches[site.id] %>>
                 <%= @report.attention_needed_breaches[site.id] %>
               </td>
+              <td class="index-table__cell attention-needed-breach-percentage">
+                <%= @report.active_sla_clients_count[site.id] %>
+
+              </td>
               <td class="index-table__cell communication-breach" data-js-count=<%= @report.communication_breaches[site.id] %>>
                 <%= @report.communication_breaches[site.id] %>
               </td>
+              <td class="index-table__cell communication-breach-percentage">
+                <%= @report.active_sla_clients[site.id] %>
+
+              </td>
               <td class="index-table__cell interaction-breach" data-js-count=<%= @report.interaction_breaches[site.id] %>>
                 <%= @report.interaction_breaches[site.id] %>
               </td>
+              <td class="index-table__cell interaction-breach-percentage">
+                <%= @report.active_sla_clients[site.id] %>
+
+              </td>
             </tr>
           <% end %>
         <% end %>
         </tbody>
       <% end %>
+      <% if @report.active_sla_clients[0] && current_user.admin? %>
```

Comment:
This is funky, but essentially when there are Unassigned clients, they come back in the grouping query as id 0. Since the admin vita partners are unscoped, the unassigned clients come back and screw up the total counts if we don't include the unassigned people. I want to update this, but I thought it was more important to get the Unassigned clients represented somehow instead of leaving the math to not add up.

ALSO, It's become really clear that I can use a partial for the rendering of the columns, but that's for a later PR.

Reply:
oh wow, that would not have occurred to me. Good catch!